finders.js

```diff
@@ -212,7 +212,33 @@ module.exports = {
     });
   },
   
-  link: fuzzyFinder('link'),
+  link: function(label) {
+    return this.addFinder({
+      find: function (elements) {
+        var links = elements.find('a').toArray().filter(function(a) {
+          var anchor = $(a);
+          var href = anchor.attr('href');
+          return typeof href !== typeof undefined &&
```

Comment:
component.find() supports options to find things by text, value, attributes etc. Maybe it would be best to extend this so you can combine a number of different search types together to get this behaviour? it might be a bit more reusable then? what do yout think @refractalize ? @joshski I am talking about elementTester here - https://github.com/featurist/browser-monkey/blob/master/elementTester.js#L74